packages/mobile/src/screens/edit-track-screen/screens/RemixSettingsScreen.tsx

```diff
@@ -233,20 +218,22 @@ export const RemixSettingsScreen = () => {
     >
       <View>
         <View style={styles.setting}>
-          {isPremium && (
-            <View style={styles.changeAvailability}>
-              <IconQuestionCircle style={styles.questionIcon} fill={neutral} />
-              <View style={styles.changeAvailabilityText}>
-                <Text>{messages.changeAvailbilityPrefix}</Text>
-                <Text>
-                  {isCollectibleGated
-                    ? messages.collectibleGated
-                    : messages.specialAccess}
-                </Text>
-                <Text>{messages.changeAvailbilitySuffix}</Text>
-              </View>
-            </View>
-          )}
+          {isPremium ? (
+            <HelpCallout
+              style={styles.changeAvailability}
+              content={
+                <View style={styles.changeAvailabilityText}>
+                  <Text>{messages.changeAvailbilityPrefix}</Text>
```

Comment:
This seems to indicate that availability needs to be changed only when it's not public, but I don't see logic for that.

Reply:
this shows info that says that we are only able to mark as remix if the track is not premium